.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Standard dockerignores for Python
+.git
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+.Python
+env
```

Comment:
could we add as well `venv`? Is a very common way to name the virtual environment folder.

Reply:
yes, agree, `venv` should be ignored as well

Reply:
@mcastellin Just now I'm seeing that only `db.sqlite3` is being ignored. However the development version of `sqlite` was renamed to `anem-per-feina.sqlite`.

https://github.com/GeeksCAT/anem-per-feina/blob/379c47df481f2aaf8813543dae2011a594777857/.env.dev.sample#L4

Reply:
@fullonic I have updated the ignore statement to include any file ending by `.sqlite3` or `.sqlite`

Reply:
Great job @mcastellin :)